commit b8e0cb62ed0f045a7ad9eb006a07e0411f6cc8d9 (refs/changes/80/1480/1)
Author: David Pacheco <dap@joyent.com>
Date:   2017-02-09T10:25:01-08:00 (2 years, 8 months ago)
    
    joyent/manta-mlive#6 mlive could be less frenetic about slow requests
    joyent/manta-mlive#7 mlive verbose logs should go to stderr

diff --git a/bin/mlive b/bin/mlive
index 959d6dd..043244f 100755
--- a/bin/mlive
+++ b/bin/mlive
@@ -126,7 +126,8 @@ function main()
 
 	args.ml_log = new mod_bunyan({
 	    'name': mlArg0,
-	    'level': process.env['LOG_LEVEL'] || 'warn'
+	    'level': process.env['LOG_LEVEL'] || 'warn',
+	    'stream': process.stderr
 	});
 
 	args.ml_manta = mod_manta.createBinClient({ 'log': args.ml_log });
@@ -398,34 +399,24 @@ LivenessTester.prototype.ensureObject = function (path, callback)
 };
 
 /*
- * Invoked frequently (10Hz by default) to fire off new requests.
+ * Invoked frequently (20Hz by default) to fire off new requests.
  */
 LivenessTester.prototype.tick = function ()
 {
 	var args = this.mlt_args;
 	var self = this;
-	var now, rq, method, rqargs;
+	var rq, method, rqargs;
 
 	mod_assertplus.ok(this.mlt_noutstanding <= args.ml_maxrqs);
 	if (this.mlt_noutstanding == args.ml_maxrqs) {
 		if (this.mlt_corked === null) {
 			this.mlt_corked = new Date();
-			fprintf(args.ml_error, '%s: corked (at %d requests)\n',
-			    this.mlt_corked.toISOString(),
-			    this.mlt_noutstanding);
 		}
 
 		return;
 	}
 
-	if (this.mlt_corked !== null) {
-		now = new Date();
-		fprintf(args.ml_error, '%s: uncorked (was corked for %d ms)\n',
-		    now.toISOString(),
-		    now.getTime() - this.mlt_corked.getTime());
-		this.mlt_corked = null;
-	}
-
+	this.mlt_corked = null;
 	this.mlt_noutstanding++;
 	rq = this.mlt_requests[this.mlt_which++];
 	this.mlt_which %= this.mlt_requests.length;
